### PR TITLE
PD-309   Update SMB Service and Multichannel content for new field

### PR DESCRIPTION
**Pull request by tonyriv3**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by Mrt134** [comment]
@tonyriv3 are these changes specific to 22.12.3?

**Comment by tonyriv3** [comment]
> @tonyriv3 are these changes specific to 22.12.3?

@Mrt134 Sorry. Yes.

File: content/SCALE/SCALETutorials/Shares/SMB/SMBMultichannel.md [hunk 1]
```diff
@@ -18,29 +18,19 @@ SMB multichannel does not function if you combine NICs into a LAGG.
...
+## Validating Multichannel Activated In Windows
 
 After you connect a client to their SMB share, open Powershell as an administrator on a client, then enter `Get-SmbMultichannelConnection`. The terminal should list multiple server IPs.
 
 ![SMBMultichannelValidateWindows1](/images/SCALE/22.12/SMBMultichannelValidateWindows1.png "Validate Multichannel")
 
-You can also enter `Get-SmbMultichannelConnection | ConvertTo-Json` and make sure `CurrentChannels` is greater than 1.
+You can also enter `Get-SmbMultichannelConnection | ConvertTo-Json` and ensure `CurrentChannels` is more than 1.
 
 ![SMBMultichannelValidateWindows2](/images/SCALE/22.12/SMBMultichannelValidateWindows2.png "Validate Multichannel")
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/Shares/SMB/SMBMultichannel.md:36]: replace with the trueimage shortcode

**Review by micjohnson777 — DISMISSED** [review]
Looks good just missed to trueimage changes.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**